tools/pubsys/src/aws/promote_ssm/mod.rs

```diff
@@ -216,8 +231,86 @@ pub(crate) async fn run(args: &Args, promote_args: &PromoteArgs) -> Result<()> {
     Ok(())
 }
 
+/// Read parameters in given file, add newly promoted parameters, and write combined parameters to
+/// the given file
+async fn write_rendered_parameters(
```

Comment:
I think `append_rendered_parameters` may be more telling, since `write` often carries a connotation of `overwrite`